### PR TITLE
[13.0][IMP] Add attribute widget field

### PR DESCRIPTION
**Pull request by qgroulard**
Here is a first attempt at letting the users specify a widget to use on attribute fields.
A 'Char' field seems the more flexible to me, but the user has to know the technical name of the widget he wants to use (which may lead to mistakes). Fortunately, entering a wrong value will not cause any error while rendering the view.

I'm looking forward to reading your suggestions.

### CODE REVIEW

File: attribute_set/__manifest__.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "Attribute Set",
-    "version": "13.0.1.0.1",
+    "version": "13.0.1.0.2",
```
Comment (simahawk) [inline attribute_set/__manifest__.py]: this is an addition, I'd go for "13.0.1.1.0"

**Review by simahawk — APPROVED** [review]
LG beside the version. Pre-approving.

**Comment by simahawk** [comment]
You'll have to rebase

**Review by HviorForgeFlow — APPROVED** [review]
Functional Review! :+1:

**Comment by HviorForgeFlow** [comment]
@qgroulard PR need rebase I guess. thanks!

**Comment by qgroulard** [comment]
@HviorForgeFlow Hmm... I think the runner has an issue, not my PR.

**Comment by HviorForgeFlow** [comment]
I thought rebasing python pre-commit would change to 3.9

**Comment by simahawk** [comment]
@HviorForgeFlow you should fix this before https://github.com/OCA/odoo-pim/pull/103 and there's one for 14 too https://github.com/OCA/odoo-pim/pull/104

**Pull request closed without merging**